ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml

```diff
@@ -68,8 +72,22 @@ tests:
     - chain: openshift-qe-control-plane
     workflow: openshift-qe-installer-aws
   timeout: 2h30m0s
-- always_run: false
-  as: conc-builds-3nodes
+- as: hybrid-control-plane-12nodes
+  cron: 0 10 * * 3
```

Comment:
@paigerube14 Regarding the cron, this profile(120 nodes) we did not run them regularly in Jenkins before, it used to be run manually once per release. If we don't have the need to escalate the priority(frequency) to run this profile, what do you think  to make it always_run: false?